qiskit/primitives/containers/pubs_result.py

```diff
@@ -23,10 +23,10 @@
 
 
 @frozen_dataclass
-class TaskResult:
-    """Result of task."""
+class PubsResult:
```

Comment:
`PubResult` might be better?

Reply:
Yes, I agree that PubResult would be better.

In general, Task <-> Pub, Tasks <-> Tasks, task <-> pub, tasks <-> pubs.

Reply:
Thanks. I updated